api/types/installers/installer.sh.tmpl

```diff
@@ -1,4 +1,15 @@
 #!/bin/sh
+
+on_ec2() {
+  EC2_STATUS=$(curl -o /dev/null -w "%{http_code}" -m5 -sS "http://169.254.169.254/latest/meta-data")
```

Comment:
Is there a way to determine the instance type without calling any API? If the API returns 500 error (which may happen), we can assume that we're on the wrong instance type and start the installation in the wrong mode.